doc/api/stream.md

```diff
@@ -495,7 +495,8 @@ added: v11.4.0
 
 * {boolean}
 
-Is `true` if it is safe to call [`writable.write()`][stream-write].
+Is `true` if it is safe to call [`writable.write()`][stream-write], which means
+the stream has not been destroyed, errored or ended.
```

Comment:
```suggestion
the stream has not been destroyed, errored, or ended.
```

Reply:
^Non-blocking, in case that wasn't obvious. (FWIW, our [Style Guide](https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md) says to use serial commas.)